src/constant.h

```diff
@@ -7,594 +7,598 @@
 // Global constants used by Moria
 // clang-format off
 
-#define MAX_UCHAR         255
-#define MAX_SHORT       32767   // maximum short/long signed ints
-#define MAX_LONG   0x7FFFFFFFL
+#pragma once
+
+#include "headers.h"
```

Comment:
I've now tried compiling and running the game on macOS, Lubuntu 14.04, and now Debian 9 (gcc 6.3), and all without issue after removing these two lines.

What errors are you getting?